jest.setup.js

```diff
@@ -0,0 +1,30 @@
+// How to catch unhandled errors when testing:
+// Reference:
+// https://chris-washington-dev.medium.com/vue-jest-properly-catch-unhandledpromiserejectionwarning-and-vue-warn-errors-in-jest-unit-tests-fcc45269146b
```

Comment:
Unfortunately, the code on this page is not open source and it's not licensed in a way we can use it. Any code copied from this page needs to be rewritten so that it is our own. Sorry.